templates/answerfile_4.2_basic.txt.j2

```diff
@@ -0,0 +1,13 @@
+{% include "./templates/basic_answerfile.txt.j2" %},
+[environment:default]
+{% if ovirt_engine_setup_provider_ovn_configure is defined and ovirt_engine_setup_provider_ovn_configure %}
+OVESETUP_OVN/ovirtProviderOvn=bool:True
+OVESETUP_OVN/ovirtProviderOvnUser=str:{{ ovirt_engine_setup_provider_ovn_username }}
+{% else %}
+OVESETUP_OVN/ovirtProviderOvn=bool:False
+{% endif %}
+{% if ovirt_engine_setup_provider_ovn_password is defined %}
+OVESETUP_OVN/ovirtProviderOvnPassword=str:{{ ovirt_engine_setup_provider_ovn_password }}
+{% else %}
+OVESETUP_OVN/ovirtProviderOvnPassword=none:None
```

Comment:
can we set the default to be the admin@internal pass instead of none?
passing none will result in a non-functional OVN provider

Reply:
@nellyc for all passwords we have default None, see for example OVESETUP_DB/password=none:None.
I think we can simply set the passwd it with the relevant parameter if you agree.

Reply:
Please consult and send a separate PR if needed, I am merging
.